### PR TITLE
data: Jainish-shah

### PR DESCRIPTION
**Pull request by Jainish-shah**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4068"></a>

### CODE REVIEW

File: data/Jainish-shah.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      "icon": "FaInstagram"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Open Source Contribution",
+      "date": "April 2021",
+      "icon": "FaGithub",
+      "description": "Contributed to FaceX",
+    },
```
Comment (Njong392) [inline data/Jainish-shah.json:46]: Oops. There seems to be a trailing comma here.
```suggestion
    }
```

File: data/Jainish-shah.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  ],
```
Comment (Njong392) [inline data/Jainish-shah.json:47]: I think there is a trailing comma here as well.
```suggestion
  ]
```

File: data/Jainish-shah.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      "url": "https://www.instagram.com/_jainish.shah_25/",
+      "icon": "FaInstagram"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Open Source Contribution",
+      "date": "April 2021",
+      "icon": "FaGithub",
+      "description": "Contributed to FaceX",
```
Comment (Njong392) [inline data/Jainish-shah.json:45]: No comma needed at the end of `description` 
```suggestion
      "description": "Contributed to FaceX"
```

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you so much for contributing to LinkFree! 🎉 Please take a look at the suggestions I have made.

File: data/Jainish-shah.json (hunk 1 above)
Comment (Jainish-shah) [reply data/Jainish-shah.json]: @Njong392  Thanks for the suggestions, I have made the changes to the file ...

**Review by Njong392 — APPROVED** [review]
Looks all good now! Thank you 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**